### ISSUE TITLE
[Programmatic Access - BOT Framework - Carousel response ]: Ensures elements with ARIA roles have all required ARIA attributes

### ISSUE THREAD

**Issue by KattaBhargav** [microsoft/BotFramework-WebChat#4343]
### User Experience:
Users who depend on screen readers will be impacted if aria rolesare not defined on the elements. Screen readers will not announce the information correctly and users will not get the proper information of the controls.

### Repro Steps:

1. Open URL: Web Chat: Full-featured bundle (microsoft.github.io)
2. Web chat bundle will open.
3. Navigate on the web chat bot and enter "carousel" keyword in Type your message box, then select send button.
4. Cards related to the keyword will appear, navigate on the cards.
5. Verify the issue.

### Actual Result:
Ensures elements with ARIA roles have all required ARIA attributes
Screen reader is announcing information as "Attachment details about image 1".

### Expected Result:
Required ARIA attribute not present: aria-level

### Issue:
Ensures elements with ARIA roles have all required ARIA attributes (aria-required-attr - https://accessibilityinsights.io/info-examples/web/aria-required-attr)
### Target application:
Web Chat: Full-featured bundle - https://microsoft.github.io/BotFramework-WebChat/01.getting-started/a.full-bundle/

### Element path:
li:nth-child(1) > .webchat__bubble.webchat__bubble--nub-on-top.webchat--css-ezira-1qo0vqe > .webchat__bubble__content > .webchat--css-ezira-1kac3oh.webchat__adaptive-card-renderer > .ac-adaptiveCard.ac-container > .ac-container[dir="ltr"] > .ac-textBlock[role="heading"]

### Snippet:
<div class="ac-textBlock" role="heading" style="overflow: hidden; font-family: Calibri, &quot;Helvetica Neue&quot;, Arial, &quot;sans-serif&quot;; font-size: 17px; color: black; font-weight: 600; text-align: start; line-height: 22.61px; white-space: nowrap; text-overflow: ellipsis; box-sizing: border-box; flex: 0 0 auto;">

How to fix:
Fix any of the following:
Required ARIA attribute not present: aria-level

MAS Reference:
MAS4.1.2- Name, Role, Value

Reference Links

- [Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-poyad%40microsoft.com%7cc11478591b5a474f6d9d08d7199726c0%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637006009300301864&sdata=ZTWYamT1yE7yko5QU35ZJLf/5ZydtDeGAKrQGUiugJs%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
- External Bug Process: If this bug belongs to the external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information, please use this [link](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx). “HCL Staff should not log any third-party external bugs. Should be routed to edad team”.
- Please reach out to the [C&AI Teams channel](https://nam06.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cv-dhti%40microsoft.com%7cf90880a914be401871f108d763233116%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637086876782361405&sdata=/ULS%2BbeqMjysCoiWHvFgAbSnuFJqxO3wtpHmlidPjbU%3D&reserved=0) for any process-related queries.

### Test Environment: 

- OS: Windows 11 Version 21H2 (OS build: 22621.160)
- Browser: Chrome version 103.0.5060.114 (Official Build) (64-bit)
- URL: https://microsoft.github.io/BotFramework-WebChat/01.getting-started/a.full-bundle/
- Tool used: Accessibility insights for web

**Comment by KattaBhargav** [comment]
#A11yMAS;#A11ySev3;#HCL;#MAS4.1.2;#Accessibility;#Win11-Chrome;#BotFrameworkWebChat-Jul22;
